File: dia_05/pessoa.py
# DIA 04: Construa uma classe pessoa
class Pessoa:
    def __init__(self, nome, endereco, rg, cpf):
        self.nome = nome
        self.endereco = endereco  
        self.rg = rg  
        self.cpf = cpf  

    # Representação do objeto
    def __rep__(self):
        return f'Pessoa: {self.nome}'

    def listar(self):
        return self.nome, self.endereco, self.rg, self.cpf  

    def herança(self):
        return "Testando a herança..."      

File: dia_06/Pessoa.cs
using System;
namespace Pessoas
{
    public class Pessoa
    {
        public Pessoa()
        {

        }

        public int Id { get; set; }
        public string Nome { get; set; }
        public string Endereco { get; set; }

        // protected: Funciona somente dentro da herança
        protected virtual string NomePersonalizado()
        {
            return $"{this.nome}";
        }

    }
}